blackbelt/dependencies.py

```diff
@@ -0,0 +1,353 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+import os
+import json
+import shutil
+import tempfile
+import subprocess
+import itertools
+
+import click
+
+
+__all__ = ['check', 'parse_dep']
+
+
+# https://github.com/davglass/license-checker#custom-format
+LICENSE_CHECKER_FORMAT_KEYS = [
+    'name', 'repository', 'dependencyPath', 'url', 'version', 'licenses',
+    'licenseText', 'licenseModified', 'licenseFile', 'path',
+]
+
+# https://github.com/davglass/license-checker#custom-format
+LICENSE_CHECKER_FORMAT = {}
+for key in LICENSE_CHECKER_FORMAT_KEYS:
+    LICENSE_CHECKER_FORMAT[key] = None
+
+# https://confluence.oraclecorp.com/confluence/display/CORPARCH/Licenses+Eligible+for+Pre-Approval+-+Internal+and+Hosted+Use
+# https://confluence.oraclecorp.com/confluence/display/CORPARCH/Licenses+Eligible+for+Pre-Approval+-+Distribution
```

Comment:
:notsureif: confluence links in OSS software...that said, blackbelt is probably kinda falling off the general usage

Reply:
Hmm. Yeah... maybe it was never really something generally used :) Do we want to close-source it? The license list is specific to Oracle, not configurable.